UefiPayloadPkg/Readme.md

```diff
@@ -0,0 +1,186 @@
+# UefiPayloadPkg
+Provide UEFI Universal Payload for different bootloader to generate EFI environment
+
+# Spec
+URL: https://docs.google.com/document/d/1WxEUlCsXpc17DkJhL3XVkOW7e_KIt_zcc9tQ1trOySg/

```

Comment:
Please change it to "URL: https://universalscalablefirmware.github.io/documentation/2_universal_payload.html"